toolkit/config.staging.yaml

```
@@ -5,10 +5,19 @@ environment:
   type: staging
   selected:
     - modules/power_model
+    # - modules/power_ops_template
 
 variables:
   modules:
+    # variables required for BOTH power_model and power_ops_template
     power_ops_type_space: "power_ops_types"
     power_ops_instance_space: "power_ops_instances"
     power_ops_models_space: "power_ops_core"
-    version: "1"
+    power_ops_version: "1"
```

```suggestion
    power_ops_data_model_version: "1"
```
Suggestion to avoid confusion with e. g. PowerOps SDK version